docs/contributors/code/getting-started-with-code-contribution.md

```diff
@@ -128,6 +128,12 @@ Port: {MYSQL_PORT_NUMBER}
 
 **Tip**: [Sequel Ace](https://sequel-ace.com/) is a useful GUI tool for accessing a MySQL database. Other tools are available and documented in this [article on accessing the WordPress database](https://developer.wordpress.org/advanced-administration/before-install/creating-database/).
 
+Alternatively, you can use phpMyAdmin to access the mySQL database. To enable access to phpMyAdmin, add the `WP_ENV_PHPMYADMIN_PORT` environment variable to the command:
+
+```bash
+WP_ENV_PHPMYADMIN_PORT=9000 npm run wp-env start
+```
+
```

Comment:
* I'd mention this earlier in the section, maybe even right after the heading. The point of phpMyAdmin is to be the "default easy" option.
* I'd also include the config option and not just the env var, since that could be a more convenient "always on" option.

A note in passing: this is the first time I see the mention of Sequel Ace, and while I appreciate the intention it's a little strange to recommend a proprietary tool that (as far as I could tell) only runs on macOS. Perhaps now with phpMyAdmin readily available we could drop the mention of Sequel Ace? I'll defer to @getdave, author of the change.

Reply:
> Perhaps now with phpMyAdmin readily available we could drop the mention of Sequel Ace?

I agree with this too.